utils/generate-command-code.py

```diff
@@ -340,12 +350,15 @@ def _doc_flags_code():
             "\"%s\"" % self.group,
             GROUPS[self.group],
             self.history_table_name(),
+            self.num_history(),
```

Comment:
now that we emit these 3 lens, we no longer need the arrays to be null-entry-terminated.
note, it may expose some use case that still used to search for that entry, and now needs to use the length.

Reply:
Done. Didn't seem to cause any problems.